mmdet3d/models/fusion_layers/point_fusion.py

```diff
@@ -293,6 +286,7 @@ def sample_single(self, img_feats, pts, img_meta):
             img_feats,
             pts,
             pts.new_tensor(img_meta['lidar2img']),
+            'LIDAR',
```

Comment:
A hardcode here might be inappropriate, can we support a new option like coord_mode in this module? Thus users can change it through config